src/components/App.js

```diff
@@ -72,11 +59,13 @@ class App extends Component {
       LinkedInLogin.getProfile();
     });
 
-    DeviceEventEmitter.addListener('linkedinGetRequest', (data) => {
-      const data = JSON.parse(data.data);
-      if (data) {
-        console.log(JSON.stringify(data));
-        console.log('data');
+    DeviceEventEmitter.addListener('linkedinGetRequest', (resData) => {
```

Comment:
What is "resData"? Does this mean response from linkedInGetRequest? If that's the case, we could just call it "response".